.github/workflows/run-renovate.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2023 TomTom NV. All rights reserved.
+
+name: Run Renovate
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '5 0 * * *'  # at 00:05 am every day
+
+permissions: {}
```

Comment:
Not needed since you declare permissions on job level.

Reply:
I am in favour of setting the minimum permission between jobs here ( even `{}` ).
this forces devs that add a job to think about what permissions said job actually needs, instead of relying on the default permissions set. 

As discussed on DM, I've add a comment there that this clear out is done on purpose.
See 130062288ac662c5fb7274ba6ebcabadfe5ae2b4